migration/migrating_3_4/about-migration.adoc

```diff
@@ -12,7 +12,8 @@ To successfully transition from {product-title} 3 to {product-title} 4, it is im
 xref:../../migration/migrating_3_4/planning-migration-3-to-4.adoc#planning-migration-3-to-4[Planning your transition]::
 Learn about the differences between {product-title} versions 3 and 4. Prior to transitioning, be sure that you have reviewed and prepared for storage, networking, logging, security, and monitoring considerations.
 
-xref:../../migration/migrating_3_4/migrating-application-workloads-3-4.adoc#migrating-application-workloads-3-4[Performing your migration]::
+xref:../../migration/migrating_3_4/migrating-application-workloads-3-4.adoc#migrating-application-workloads-3-4[Migration concepts and prerequisites]::
```

Comment:
@apinnick I think I'd like to keep this named as it was. It was never an exact name to the heading. It was more for trying to differentiate that the above section I added was for planning, and then your docs were for actually performing it. Or would "Analyzing your migration" be more accurate now due to the name changes you had in a PR a few weeks ago?

Reply:
I see the distinction you're making. I'll revert to your original text because it makes more sense than "analyzing the migration". The assembly is more about the tools for performing the migration.